bigbluebutton-html5/imports/ui/components/poll/component.jsx

```diff
@@ -235,13 +287,13 @@ class Poll extends Component {
   }
 
   componentDidUpdate() {
-    const { amIPresenter, layoutContextDispatch, sidebarContentPanel } = this.props;
+    const { amIPresenter, layoutContextDispatch } = this.props;
 
     if (Session.equals('resetPollPanel', true)) {
       this.handleBackClick();
     }
 
-    if (!amIPresenter && sidebarContentPanel === PANELS.POLL) {
```

Comment:
Removing these parts brings back the issue fixed by #14175

Reply:
There was bug in poll due to this lines of code  following gif show it 
![polling-context-bug](https://user-images.githubusercontent.com/101873058/163786971-6fb2fd7b-4be3-44fd-af85-dd36119c6d43.gif)